pset-iterators.js

```diff
@@ -65,6 +70,16 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 2:')
 
 // Add your code below this line
+const presidentsNamedJames = presidents.filter((president) => {
+  return president.president.includes('James')
+}).map(function (president) {
+  return president.president
+  })
+
+
+console.log(presidentsNamedJames)
```

Comment:
@jct117 Great work here on this solution to look up the presidents whose name includes 'James' using the filter method.
One Suggestion
-For this example, you could remove the map method because for this example you were to return an Array of objects and the filter method without the map will give you that outcome.

Take a look at the alternate solutions [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L62)